packages/terra-cli/package.json

```diff
@@ -28,9 +30,16 @@
   },
   "homepage": "https://github.com/cerner/terra-toolkit",
   "scripts": {
-    "lint": "eslint --ext .js,.jsx ."
+    "compile": "babel src --out-dir lib --copy-files --ignore src/utils",
+    "jest": "jest --config ../../jest.config.js",
+    "lint": "eslint --ext .js,.jsx .",
+    "test": "npm run jest"
   },
   "dependencies": {
+    "fs-extra": "^9.0.1",
+    "lockfile": "^1.0.4",
+    "npmlog": "^4.1.2",
+    "yargs": "^14.2.2"
```

Comment:
Out of curiosity, why not install yargs v16?

Reply:
Good catch. I'm not sure how I got this version other than maybe starting from lerna's starting point. The breaking changes seem to be pretty straightforward, so I'll update.

Reply:
Updated: https://github.com/cerner/terra-toolkit/pull/460/commits/5314365d74c3ef7d44104a23b5d3143723176f6d